deploy/docker-swarm/clickhouse-setup/otel-collector-metrics-config.yaml

```diff
@@ -9,12 +9,12 @@ receivers:
     config:
       scrape_configs:
         - job_name: "otel-collector"
-          scrape_interval: 30s
+          scrape_interval: 60s
```

Comment:
Looks good to me, but with new metrics builder the graph plot would be slightly different than before.

Related to this: https://github.com/SigNoz/engineering-pod/issues/487

Reply:
Yeah, I agree. The above issue should be useful later.